openstack/networking/v2/extensions/fwaas_v2/rules/requests.go

```diff
@@ -39,6 +40,74 @@ const (
 	ActionReject Action = "reject"
 )
 
+// ListOptsBuilder allows extensions to add additional parameters to the
+// List request.
+type ListOptsBuilder interface {
+	ToRuleListQuery() (string, error)
+}
+
+// ListOpts allows the filtering and sorting of paginated collections through
+// the API. Filtering is achieved by passing in struct field values that map to
+// the Firewall rule attributes you want to see returned. SortKey allows you to
+// sort by a particular firewall rule attribute. SortDir sets the direction, and is
+// either `asc' or `desc'. Marker and Limit are used for pagination.
+type ListOpts struct {
+	TenantID             string   `q:"tenant_id"`
+	Name                 string   `q:"name"`
+	Description          string   `q:"description"`
+	Protocol             Protocol `q:"protocol"`
+	Action               Action   `q:"action"`
+	IPVersion            int      `q:"ip_version"`
+	SourceIPAddress      string   `q:"source_ip_address"`
+	DestinationIPAddress string   `q:"destination_ip_address"`
+	SourcePort           string   `q:"source_port"`
+	DestinationPort      string   `q:"destination_port"`
+	Enabled              bool     `q:"enabled"`
+	ID                   string   `q:"id"`
+	Shared               bool     `q:"shared"`
```

Comment:
Same here

Reply:
Should be resolved in commit 4e8d6bc7085d63aea8479da88fba68c6a2a985a6